core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -111,6 +116,8 @@ export class NavigationComponent implements OnInit {
       });
 
     this.registerWorkflowMetadataDisplayRefresh();
+
+    this.handleDisableOperatorIconChange();
```

Comment:
maybe name it `registerDisableOperatorIconChange`? so that you register a handler for an event.

Reply:
in many places we use `handleXXX` for this kind of functions